src/components/Home/Home.js

```diff
@@ -0,0 +1,125 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import Coin from '../coin/coin';
+import './home.css';
+
+const CoinsList = ({ coins }) => {
+  const coinItems = coins.map((coin) => (
+    <Coin coin={coin} key={Math.random()} />
+  ));
+
+  return <ul className="crypto-list">{coinItems}</ul>;
+};
+
+CoinsList.propTypes = {
+  coins: PropTypes.oneOfType([PropTypes.array, PropTypes.object]).isRequired,
+};
+
+const Error = ({ error }) => (
+  <div className="error-container" data-testid="error-container">
+    <span>{error}</span>
+  </div>
```

Comment:
Nice job so far😉. Let's try to ensure our project matches the given design. take note of the highlighted components and ensure the spacings are consistent as in the design. also, do well to stay consistent with the color as done in design 😉. you should use a different shade of blue, instead of making use of white. do well to implement that.


Your design
<img width="432" alt="Screenshot 2023-09-08 at 09 08 17" src="https://github.com/riskmaris/bitcoin-app/assets/64643995/77144e06-d483-4f7c-b40d-03f3119b87bf">

Figma design
<img width="430" alt="image" src="https://github.com/riskmaris/bitcoin-app/assets/64643995/5eb10629-cdba-41a1-9fdd-4c8346c46209">